guides/release/using-typescript/additional-resources/index.md

```diff
@@ -0,0 +1,3 @@
+---
+redirect: using-typescript/additional-resources/legacy
```

Comment:
why do we have a redirect for a page that hasn't launched yet? What I said before about "never breaking a link" is true, but we don't need to do that for things that are not released yet 🤔